make-webpack-config.js

```diff
@@ -54,13 +54,12 @@ module.exports = function(rules, options) {
 
   if( specialOptions.separateStylesheets ) {
     plugins.push(new ExtractTextPlugin({
-      filename: '[name].css' + (specialOptions.longTermCaching ? '?[contenthash]' : ''),
+      filename: "[name].css" + (specialOptions.longTermCaching ? "?[contenthash]" : ""),
```

Comment:
@owenconti and I had an out-of-band conversation, and settled on hardcoding `swagger-ui.css`.